domains/Discussions/Controllers/QuestionsStoreController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+
+class QuestionsStoreController extends Controller
+{
+    public function __invoke(Request $request): Response
+    {
+        $this->validate($request, [
+            'title' => ['required', 'string', 'max:255'],
+            'description' => ['required', 'string'],
+        ]);
+
+        (new Question())
+            ->forceFill([
+                'author_id' => Auth::id(),
```

Comment:
We should use the `$request` class to et the authenticated user's id.

Reply:
done